qemu/tests/cfg/snp_basic_config.cfg

```diff
@@ -0,0 +1,34 @@
+- snp_basic_config:
+    type = snp_basic_config
+    kill_vm = yes
+    login_timeout = 240
+    start_vm = no
+    image_snapshot = yes
```

Comment:
I guess you want a snapshot here in order to keep the original image(image1) clean after your testing without any cleanup works, right? Have you checked if it works as expected?

Reply:
Yes, it works with single and multi vms, will image file e.g. copy vl_avocado-vt-vm1_image1.qcow2